### PR TITLE
Update update cni conf

### PR DESCRIPTION
**Pull request by nagiesek**
Fixes the cni confs to use dns Runtime from k8s. Download correct binaries

### CODE REVIEW

**Comment by madhanrm** [comment]
Is this ready for merge or pending validation

**Comment by nagiesek** [comment]
> 
> 
> Is this ready for merge or pending validation
I'll do a full validation now.

File: Kubernetes/windows/helper.v2.psm1 [hunk 1]
```diff
@@ -710,12 +698,11 @@ Update-CNIConfig
             "cniVersion": "0.2.0",
             "name": "<NetworkMode>",
             "type": "flannel",
+            "capabilities": {
+                "dns" : true
```
Comment (madhanrm) [inline Kubernetes/windows/helper.v2.psm1]: can you also add portmapping capabilities ?

**Comment by madhanrm** [comment]
Am going to merge this now

File: Kubernetes/windows/helper.v2.psm1 (hunk 1 above)
Comment (nagiesek) [reply Kubernetes/windows/helper.v2.psm1]: Will do in followup

Comment (nagiesek) [reply Kubernetes/windows/helper.v2.psm1]: Portmapping isn't implemented in the containernetworking/plugins binaries.

**Pull request merged**